Scripts/DCS-BIOS/lib/modules/aircraft_modules/Ka-50.lua

```diff
@@ -0,0 +1,917 @@
+module("Ka-50", package.seeall)
+
+local Control = require("Control")
+local ControlType = require("ControlType")
+local IntegerOutput = require("IntegerOutput")
+local Module = require("Module")
+local PhysicalVariant = require("PhysicalVariant")
+local Suffix = require("Suffix")
+
+--- @class Ka_50 : Module
+local Ka_50 = Module:new("Ka-50", 0x1800, { "Ka-50", "Ka-50_3" })
+
+--overhaul by WarLord v2.0b
+
+-- Hide Stick Arg# 1005
+
+--- Defines a push button LED
+--- @param self Module the calling module
+--- @param identifier string the unique identifier for the control
+--- @param arg_number integer the dcs argument number
+--- @param category string the category in which the control should appear
+--- @param description string additional information about the control
+--- @return Control control the control which was added to the module
+local function definePushButtonLed(self, identifier, arg_number, category, description)
+	local function round(n)
+		return n % 1 >= 0.5 and math.ceil(n) or math.floor(n)
+	end
+
+	local max_value = 1
+	local alloc = self:allocateInt(max_value)
+	assert(alloc.shiftBy ~= nil)
+	self:addExportHook(function(dev0)
+		alloc:setValue(round(dev0:get_argument_value(arg_number) * 10) % 2)
```

Comment:
Were you able to test and see what this value looks like in insight (or just via logging)?